newswires/client/src/WireItemList.tsx

```diff
@@ -64,17 +58,15 @@ export const WireItemList = ({
 					),
 				)}
 			</ul>
-			{wires.length < totalCount && (
```

Comment:
nb. removing the `{wires.length < totalCount && () }` condition means that the 'load more' button will always show, even if there's nothing more to load for the given search (except in the case where there are just no results in the first place)

I think this is reasonable for now -- nothing breaks if the button is pressed when there's nothing more to load, and from what we know of current user behaviour it doesn't seem likely that people are often getting to the end of a list of results like that.

Something to address in a more robust way down the line though, when we know what the longer-term plan is for result counts